pkgs/applications/science/math/gap/default.nix

```diff
@@ -23,27 +24,28 @@ let
   # packages absolutely required for gap to start
   # `*` represents the version where applicable
   requiredPackages = [
-    "GAPDoc-*"
-    "primgrp-*"
-    "SmallGrp-*"
+    "gapdoc"
+    "primgrp"
+    "smallgrp"
     "transgrp"
   ];
   # packages autoloaded by default if available
   autoloadedPackages = [
     "atlasrep"
-    "autpgrp-*"
-    "alnuth-*"
-    "crisp-*"
-    "ctbllib-*"
-    "FactInt-*"
+    "autpgrp"
+    "alnuth"
```

Comment:
I am not sure if this is relevant, but: to be actually useful, `alnuth` needs `pari-gp`. But it only needs a path to the `gp` executable.

To be clear, this does not affect functionality of basic GAP, just when someone starts to interact with this particular part of it...

I am not sure what conclusion to draw from this; perhaps none; I just wanted to make sure you are aware (perhaps you already are)

Reply:
I was not, thanks for mentioning it! I will add pari as a dependency.